packages/eslint-plugin/tests/rules/no-confusing-void-expression.test.ts

```diff
@@ -119,6 +119,34 @@ function cool(input: string) {
 }
       `,
     },
+    {
+      options: [{ ignoreVoidInVoid: true }],
+      code: `
+function returnsVoid(): void {}
+
+function test1(): void {
+  return returnsVoid(); // should be fine
+}
+      `,
```

Comment:
[testing] The rest test cases use `console.log` to produce the `void` type, let's use `return console.log('foo')` to be consistent with other test cases? Also, using `console.log` would shorten all cases by two lines, so I think the tests would be more readable. WDYT?

Reply:
i think that's good idea!